reporting/src/request/index.js

```diff
@@ -483,24 +441,30 @@ export default class RequestReporter {
     }
     // shouldBlockCookie
     if (
-      (!this.checkIsWhitelisted(state) &&
-        this.isCookieEnabled(state) &&
-        !this.config.paused) === false
+      (!this.checkIsWhitelisted(state) && this.isCookieEnabled(state)) === false
     ) {
       state.incrementStat('bad_cookie_sent');
       return response.toWebRequestResponse();
     }
 
-    this.#reportTrackerInteraction('cookie-removed', state);
-
-    // blockCookie
-    state.incrementStat('cookie_blocked');
-    state.incrementStat('cookie_block_tp1');
-    response.modifyHeader('Cookie', '');
-    if (this.config.sendAntiTrackingHeader) {
-      response.modifyHeader(this.config.cliqzHeader, ' ');
+    if (this.dryRunMode) {
```

Comment:
I don't see the dryRunMode parameter passed in. Can we take the setup from https://github.com/whotracksme/webextension-packages/pull/142/files?